lib/mp4/transmuxer.js

```diff
@@ -1100,23 +1180,39 @@ Transmuxer = function(options) {
 
   // feed incoming data to the front of the parsing pipeline
   this.push = function(data) {
+    if (hasFlushed) {
```

Comment:
It allows us to support, seamlessly, situations where we have audio in both (TS; AAC) formats in one manifest. Might as well support that now and not worry about it in the future.